### DISCUSSION TITLE
Conditional dotenv-filename?

### DISCUSSION THREAD

**Discussion by Darkle** [casey/just#1732] [Q&A]
Hi, is it possible to do a conditional dotenv-filename? I was hoping to do something like the following:
```
isE2ETest := "false"

set dotenv-filename := if isE2ETest == "true" { "../../test/.env" } else { "../../.env" }
```
but I get an error message of "Expected string, but found identifier".

**Comment by afeldkamp** [comment]
...or even just let us use a variable in the definition, like:
```
set dotenv-filename := ".{{ current_project }}.env"
```

**Comment by casey** [comment]
#1849, in progress, performs expansion of variables and `~` inside import paths. I think it would be reasonable to extend it to perform such expansion on the value of `dotenv-filename`. Although, this does not support arbitrary just expressions.
